### PR TITLE
Stories/es report updates

### PR DESCRIPTION
**Pull request by eanders**

### CODE REVIEW

File: app/assets/javascripts/warehouse_reports/rrh/makeup.js.coffee [hunk 1]
```diff
@@ -38,9 +38,9 @@ class App.WarehouseReports.Rrh.Makeup
           ticks: 4
       legend:
         show: true
-        position: 'right'
+        position: 'bottom'
```
Comment (awisema) [inline app/assets/javascripts/warehouse_reports/rrh/makeup.js.coffee:41]: This didn't solve the problem for me -- do we need to to estimate the size somehow, or is this just bad data for me (I see a NaN)?

image

File: app/models/warehouse_report/outcomes/base.rb [hunk 2]
```diff
@@ -308,8 +308,7 @@ def destinations
     end
   end
 
-  def destination_bucket(client_id, dest_id)
-    return 'returned to shelter' if returns_to_shelter_after_ph.keys.include?(client_id)
+  def destination_bucket(dest_id)
```
Comment (awisema) [inline app/models/warehouse_report/outcomes/base.rb:311]: With this removed from here, should _A return to homelessness is defined as any subsequent night in an Emergency Shelter or on the street, either directly after exiting, or after exiting to a permanent housing destination._ be removed from the Destination at Exit text?

**Review by awisema — CHANGES_REQUESTED** [review]

File: app/assets/javascripts/warehouse_reports/rrh/makeup.js.coffee (hunk 1 above)
Comment (eanders) [reply app/assets/javascripts/warehouse_reports/rrh/makeup.js.coffee:41]: I _think_ that's a data problem, I don't have quite so many projects, but with a sizable number it seems to render fine.  Happy to up the size too.

Comment (eanders) [reply app/assets/javascripts/warehouse_reports/rrh/makeup.js.coffee:41]: Screen Shot 2021-04-05 at 4 47 36 PM
Screen Shot 2021-04-05 at 4 48 01 PM

**Review by awisema — APPROVED** [review]

**Pull request merged**